dashboard/app/models/course_offering.rb

```diff
@@ -34,6 +34,8 @@ class CourseOffering < ApplicationRecord
   has_many :course_versions, -> {where(content_root_type: ['UnitGroup', 'Unit'])}
   belongs_to :self_paced_pl_course_offering, class_name: 'CourseOffering', optional: true
 
+  has_and_belongs_to_many :pd_workshops, class_name: 'Pd::Workshop', join_table: :course_offerings_pd_workshops, association_foreign_key: 'pd_workshop_id'
```

Comment:
nit: are all these parameters needed?

Reply:
Offline discussion: We felt some of the parameters had to stay to handle the namespace difference of `workshop` vs. `pd_workshop`, but the `association_foreign_key: 'course_offering_id'` line wasn't necessary in `workshop.rb`